tox.ini

```diff
@@ -5,7 +5,6 @@ skipsdist = True
 [testenv]
 deps =
   -rrequirements.txt
-  git+https://github.com/ioggstream/docutils@docsitalia
```

Comment:
Le modifiche sono state integrate in docutils, quindi non dovrebb'essere più necessario usare la versione patchata.

Reply:
@ioggstream cosa intendi per versione patchata?

Reply:
Prima per far compilare il latex era necessario usare una versione modificata di docutils presa dal mio repository. Ora le modifiche nostre sono state integrate nella versione ufficiale, quindi non c'è più bisogno di usare la versione del mio repository ma è possibile usare la versione ufficiale. @vintra73